adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -294,13 +297,16 @@ private AuthenticationResult acquireTokenSilentFlow(final AuthenticationRequest
     }
 
     /**
-     * Try acquire token silent locally.
-     */
+    * Try acquire token silent locally.
+    */
     private AuthenticationResult tryAcquireTokenSilentLocally(final AuthenticationRequest authenticationRequest)
             throws AuthenticationException {
         Logger.v(TAG, "Try to silently get token from local cache.");
         final AcquireTokenSilentHandler acquireTokenSilentHandler = new AcquireTokenSilentHandler(mContext,
                 authenticationRequest, mTokenCacheAccessor);
+        if(mAuthContext.getExtendedLifetimeEnabled()) {
```

Comment:
We cannot move this condition into setOutageModeIsOn(). Because setOutageModeIsOn() belong to class AcquireTokenSilentHandler, and AcquireTokenSilentHandler does not contain the property AuthenticationContext object.